### PR TITLE
[FARM-502] Added edit and delete profile on farm details

### PR DESCRIPTION
**Pull request by davidfranquet**
[FARM-502]

#### 📲 What

Added edit and delete profile on farm details and hide the "more options" button when comes from onboarding chat flow

#### 🤔 Why
		
Needed for MVP

#### 👀 See
		
ezgif com-video-to-gif-17

**Coming from On boarding / new profile chat flow (hide 'more options' button)**

		 
#### ✅ Acceptance criteria

- [ ] Design Review for UI with BA completed. 
- [ ] Manually tested and verified on Android device/emulator (min. Lollipop 5.1)
- [ ] Showcase video / automation test
- [ ] Passing all tests
- [ ] Rebased/merged with latest changes from development and re-tested
- [ ] Removed unsed comments. TODOs must have JIRA for future work.

#### Coding Standards Checklist
- [ ] unit tests 80% coverage on testable code (functions, methods, class)
- [ ] (Architecture) redux state, reducers, actions, middleware defined under lib/redux
- [ ] ui elements defined under lib/ui. 
- [ ] Strings ready for localisation (e.g. defined in lib/utils/strings.dart)
- [ ] Assets (images/icons path) defined in lib/utils/assets.dart
- [ ] Colors defined in lib/utils/colors.dart
- [ ] Dimensions (ie margins padding) defined in lib/utils/dimens.dart
- [ ] TextStyles defined in lib/utils/styles.dart

Recommended Style Guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

#### 🕵️‍♂️ How to test

Notes for QA

#### Reviewers

@farmsmart/amido @farmsmart/wamf

### CODE REVIEW

File: lib/ui/profile/Profile.dart [hunk 1]
```diff
@@ -525,6 +525,8 @@ class Profile extends StatelessWidget {
       context,
       FarmDetails(
         viewModel: FarmDetailsViewModel(
+          removeProfile: viewModel.remove,
+          editProfile: () => viewModel.newAccountFlow.run(context),
```
Comment (davidfranquet) [inline lib/ui/profile/Profile.dart:529]: We should do a new chat flow to edit the profile, right now is creating a new one. Will be done in another PR
https://amidodevelopment.atlassian.net/browse/FARM-513

Comment (LeeMatthewHiggins) [reply lib/ui/profile/Profile.dart:529]: yes for sure

**Review by LeeMatthewHiggins — APPROVED** [review]

**Review by alexanishchenko88 — APPROVED** [review]
